### ISSUE TITLE
Build NuGet package directly

### ISSUE THREAD

**Issue by punker76** [hardcodet/wpf-notifyicon#3]
After converting to the new csproj format we can build now the NuGet package directly with it.

**Comment by Suplanus** [comment]
Is it possible to strong name the NuGet? You know about the problems 🦄

**Comment by Lakritzator** [comment]
@punker76 as I just commented in the issue #4 I already can build the package directly.
Someone should check if the content is okay like that.

About strong naming, I guess we should be able to do so, but not from a security point of view.

**Comment by Lakritzator** [comment]
I finally managed to free up some time, and fix the issue that I couldn't get the build working.
The issue was pretty much this: https://github.com/dotnet/msbuild/issues/4704
But I had to change the build, to use msbuild instead of "dotnet build" as I couldn't get it running for net45
I don't care about it, and would suggest to drop the support to make this easier to maintain... but it's working.

I also fixed some other issues and merged a PR, maybe we could talk about a release?
